pilot/pkg/networking/plugin/mixer/mixer.go

```diff
@@ -697,3 +700,17 @@ func attrsCopy(attrs attributes) attributes {
 	}
 	return out
 }
+
+func getQuotaSpec(in *plugin.InputParams, hostname host.Name, isPolicyCheckDisabled bool) []*mccpb.QuotaSpec {
```

Comment:
Consider make it reusable by inbound route

Reply:
@hzxuzhonghu is it possible to approve this the way it is?
Otherwise please “request-changes” and block the pr.